cobra/__init__.py

```diff
@@ -1158,25 +1160,105 @@ def initSocketBuilder(host,port):
         socket_builders[ (host,port) ] = builder
     return builder
 
-def startCobraServer(host="", port=COBRA_PORT):
-    global daemon
-    if daemon is None:
-        daemon = CobraDaemon(host,port)
-        daemon.fireThread()
-    return daemon
+def getCobraDaemon(host="", port=COBRA_PORT, sslca=None, sslcrt=None, sslkey=None, msgpack=True, create=True, wait=False):
```

Comment:
Is there any particular reason we need these now that users can just instantiate the DaemonManger class and go from there? These feel kinda redundant now. Or are these kept around mostly for backwards compatibility?

Also, do we need to automatically setup an instance for them like we do on line 1285 of this patch? I feel like it'd be cleaner as a context manager helper.

Reply:
this is backwards compat